ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_shared_utils/misc.py

```diff
@@ -154,6 +154,14 @@ def evpn_prevent_readvertise_to_server(self: SharedUtils) -> bool:
     def dc_name(self: SharedUtils) -> str | None:
         return get(self.hostvars, "dc_name")
 
+    @cached_property
+    def fabric_name(self: SharedUtils) -> str | None:
+        return get(self.hostvars, "fabric_name")
```

Comment:
We will not run if fabric_name is not set, so you can add `required=True` to the get and then remove the | None